UC4-chaincode/src/test/java/de/upb/cs/uc4/chaincode/TestCreationBase.java

```diff
@@ -0,0 +1,51 @@
+package de.upb.cs.uc4.chaincode;
+
+
+import com.google.gson.reflect.TypeToken;
+import de.upb.cs.uc4.chaincode.model.JsonIOTest;
+import de.upb.cs.uc4.chaincode.util.GsonWrapper;
+import org.junit.jupiter.api.DynamicTest;
+import org.junit.jupiter.api.TestFactory;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.lang.reflect.Type;
+import java.util.ArrayList;
+import java.util.List;
+
+public abstract class TestCreationBase {
+
+    abstract DynamicTest CreateTest(JsonIOTest test);
+
+    abstract String GetTestConfigDir();
```

Comment:
```suggestion
    abstract String getTestConfigDir();
```
This is the java way, i think...